nextjs-end/components/Header.jsx

```diff
@@ -0,0 +1,108 @@
+"use client";
+
+import React, { useState, useEffect } from "react";
+import Link from "next/link";
+import {
+	signInWithGoogle,
+	signOut,
+	onAuthStateChanged,
+} from "@/lib/firebase/auth.js";
+import { addFakeRestaurantsAndReviews } from "@/lib/firebase/firestore.js";
+
+// If the user logs in on the client, we send the user data to the server
+// Sending a user payload to this endpoint will create a persistent cookie
+async function handleUserSession(user = {}) {
+	return fetch("http://localhost:3000/api", {
```

Comment:
Can we add a comment explaining why we're hard coding this endpoint?

Reply:
Oh good catch, it didn't need to be absolute. Fixed!